targets/uefi-amd64.json

```diff
@@ -0,0 +1,39 @@
+{
+  "build-tags": ["uefi", "baremetal", "linux", "amd64"],
+  "llvm-target": "x86_64-unknown-windows-gnu",
+  "goos": "linux",
+  "goarch": "amd64",
+  "gc": "precise",
+  "linker": "ld.lld",
+  "libc": "picolibc",
+  "scheduler": "tasks",
+  "automatic-stack-size": false,
+  "default-stack-size": 65536,
+  "cflags": [
+    "-Werror",
+    "-fshort-enums",
+    "-fomit-frame-pointer",
+    "-fno-exceptions", "-fno-unwind-tables", "-fno-asynchronous-unwind-tables",
+    "-ffunction-sections", "-fdata-sections",
+    "-ffreestanding",
+    "-fshort-wchar",
+    "-mno-red-zone"
+  ],
```

Comment:
I would suggest adding `"uefi-amd64"` to builder/builder_test.go and running the tests. I suspect there will be a difference in the `"features"` JSON key (which is relevant for the ABI among other things).